Assets/Scripts/BlockBehaviour.cs

```diff
@@ -10,10 +10,15 @@ public class BlockBehaviour : MonoBehaviour
 
     [Range(0f, 1f)]
     public float SkinToLengthRatio = 0.1f;
-    public float Speed = 1f;
+
+    private float CurrentSpeed;
+    public float InitialSpeed = 1f;
+    public float AccelerationFactor = 10f;

```

Comment:
I would suggest using Acceleration instead of a factor. Speed Will be update as such:
```cs
CurrentSpeed += Acceleration * Time.deltaTime
```
To be done every frame.